modules/vaos/app/services/vaos/v2/appointments_presentation_filter.rb

```diff
@@ -9,28 +9,13 @@ def initialize
       end
 
       def user_facing?(appointment)
-        return true if valid_appointment?(appointment) &&
-                       (presentable_upcoming_appointment?(appointment) ||
-                         presentable_past_appointment?(appointment) ||
-                         presentable_cancelled_appointment?(appointment))
+        return true if valid_appointment?(appointment)
 
         presentable_requested_appointment?(appointment)
       end
 
       private
 
-      def presentable_upcoming_appointment?(appointment)
-        appointment[:start] >= @now
-      end
-
-      def presentable_past_appointment?(appointment)
-        appointment[:start] < @now && appointment[:status] != 'cancelled'
```

Comment:
@simiadebowale the original ticket asked that this condition be updated so that `appointment[:status] != 'cancelled'` is removed. However, this ultimately reduces the logic into just checking if it's a valid appointment or presentable request. Just wanted to double check if this is the intended result.